Warning on 'redeclared inline' methods in SharingTagSet.h and SharingRegionTagSet.h and linker failing (Windows OS)

Hi guys.
Trying to compile my project which references INET (Windows OS, v4.4.x INET branch, commit 8d683bdde6), and I get many warnings on `SharingTagSet.h` file like this one:
```
../../inet/src/inet/common/packet/tag/SharingTagSet.h:143:38: warning: 'inet::SharingTagSet::operator=' redeclared inline; 'dllimport' attribute ignored [-Wignored-attributes]
inline SharingTagSet& SharingTagSet::operator=(const SharingTagSet& other)
```
and many warnings on the `SharingRegionTagSet.h` file like this one:
```
../../inet/src/inet/common/packet/tag/SharingRegionTagSet.h:284:50: warning: 'inet::SharingRegionTagSet::operator=' redeclared inline; 'dllimport' attribute ignored [-Wignored-attributes]
inline SharingRegionTagSet& SharingRegionTagSet::operator=(const SharingRegionTagSet& other)
```

These warnings are pretty annoying as they pop up at each module compilation.

Worst thing is that my project seems not linking because of them. In fact the linker gives me the following error on the object of an header which extends `FieldsChunk`:
```
lld-link: error: duplicate symbol: inet::SharingRegionTagSet::operator=(inet::SharingRegionTagSet const&)
>>> defined at ../../inet/src/inet/common/packet/tag/SharingRegionTagSet.h:285
>>>            <my_message>_m.o
>>> defined at libINET_dbg.dll
```

Can you acknowledge there's an issue there?
In any case, I opened a pull request to solve the issue. Looking forward to feedbacks.

Bye
Luca

--- a/src/inet/common/packet/tag/SharingRegionTagSet.h
+++ b/src/inet/common/packet/tag/SharingRegionTagSet.h
@@ -123,7 +123,7 @@ class INET_API SharingRegionTagSet : public cObject
     Ptr<SharedVector<RegionTag<TagBase>>> regionTags;
 
   protected:
-    void setTag(int index, const Ptr<const TagBase>& tag);
+    inline void setTag(int index, const Ptr<const TagBase>& tag);
     void addTag(b offset, b length, const Ptr<const TagBase>& tag);
     std::vector<SharingRegionTagSet::RegionTag<TagBase>> addTagsWhereAbsent(const std::type_info& typeInfo, b offset, b length, const Ptr<const TagBase>& tag);
     const Ptr<TagBase> removeTag(int index);
@@ -139,7 +139,7 @@ class INET_API SharingRegionTagSet : public cObject
 
     void ensureTagsVectorAllocated();
     void prepareTagsVectorForUpdate();
-    void sortTagsVector();
+    inline void sortTagsVector();
 
   public:
     /** @name Constructors and operators */
@@ -148,8 +148,8 @@ class INET_API SharingRegionTagSet : public cObject
     SharingRegionTagSet(const SharingRegionTagSet& other) : regionTags(other.regionTags) {}
     SharingRegionTagSet(SharingRegionTagSet&& other) : regionTags(other.regionTags) { other.regionTags = nullptr; }
 
-    SharingRegionTagSet& operator=(const SharingRegionTagSet& other);
-    SharingRegionTagSet& operator=(SharingRegionTagSet&& other);
+    inline SharingRegionTagSet& operator=(const SharingRegionTagSet& other);
+    inline SharingRegionTagSet& operator=(SharingRegionTagSet&& other);
 
     virtual void parsimPack(cCommBuffer *buffer) const override;
     virtual void parsimUnpack(cCommBuffer *buffer) override;
@@ -160,27 +160,27 @@ class INET_API SharingRegionTagSet : public cObject
     /**
      * Returns the number of tags.
      */
-    int getNumTags() const;
+    inline int getNumTags() const;
 
     /**
      * Returns the shared tag at the given index.
      */
-    const Ptr<const TagBase>& getTag(int index) const;
+    inline const Ptr<const TagBase>& getTag(int index) const;
 
     /**
      * Returns the exclusively owned tag at the given index for update.
      */
-    const Ptr<TagBase> getTagForUpdate(int index);
+    inline const Ptr<TagBase> getTagForUpdate(int index);
 
     /**
      * Returns the shared region tag at the given index.
      */
-    const RegionTag<TagBase>& getRegionTag(int index) const;
+    inline const RegionTag<TagBase>& getRegionTag(int index) const;
 
     /**
      * Returns the exclusively owned region tag at the given index.
      */
-    const RegionTag<TagBase> getRegionTagForUpdate(int index);
+    inline const RegionTag<TagBase> getRegionTagForUpdate(int index);
 
     /**
      * Clears the set of tags in the given region.

--- a/src/inet/common/packet/tag/SharingTagSet.h
+++ b/src/inet/common/packet/tag/SharingTagSet.h
@@ -40,7 +40,7 @@ class INET_API SharingTagSet : public cObject
     Ptr<SharedVector<Ptr<const TagBase>>> tags;
 
   protected:
-    void setTag(int index, const Ptr<const TagBase>& tag);
+    inline void setTag(int index, const Ptr<const TagBase>& tag);
     void addTag(const Ptr<const TagBase>& tag);
     const Ptr<TagBase> removeTag(int index);
 
@@ -57,8 +57,8 @@ class INET_API SharingTagSet : public cObject
     SharingTagSet(const SharingTagSet& other) : tags(other.tags) {}
     SharingTagSet(SharingTagSet&& other) : tags(other.tags) { other.tags = nullptr; }
 
-    SharingTagSet& operator=(const SharingTagSet& other);
-    SharingTagSet& operator=(SharingTagSet&& other);
+    inline SharingTagSet& operator=(const SharingTagSet& other);
+    inline SharingTagSet& operator=(SharingTagSet&& other);
 
     virtual void parsimPack(cCommBuffer *buffer) const override;
     virtual void parsimUnpack(cCommBuffer *buffer) override;

